### DISCUSSION TITLE
Remove item

### DISCUSSION THREAD

**Discussion by loneranger1982** [sangcomz/StickyTimeLine#35] [General]
Hi 

thanks for this great extension, i am just trying to remove an item from the recycler view, i am currently removing the item then notifyitemRemoved() but am getting array index out of bounds any tips how to remove an item ?

@sangcomz

**Comment by sangcomz** [comment]
Hi, @loneranger1982 

I didn't understand the question very well. Are you saying that removing will cause a crash? If yes, could you please share the log as well?
